examples/bri-3/docs/nats/nats-configuration.md

```diff
@@ -0,0 +1,59 @@
+# NATS configuration
+
+We are relying on simple user authorization. Each BpiSubject is represented with a single user in Nats. Each Nats user is authorized for specific topics through the permissions map in nat server congif file. Currenly, we assume that the Bpi Admin will manage these users and their permssions. [Link](    https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro) to Nats docs.
```

Comment:
please clarify this is authz for nats only, to not confuse with general authz

also typos:
- congif
- currenly
- BPi
- `docker run -p 4222:4222 -d nats:alpine` -  (no need for - at the end, might be confusing or even make command invalid)
- docker cp <donwload_location>
- generaate 

something tells me there are more

Reply:
Fixed